sopel/module.py

```diff
@@ -248,8 +248,10 @@ def add_attribute(function):
     return add_attribute
 
 
-def rate(value):
-    """Decorate a function to limit how often a single user may trigger it.
+def rate(user_rate, channel_rate=0, global_rate=0):
```

Comment:
It would, however, allow using the decorator like `@rate(channel_rate=300)` to limit a command per-channel without having to specify a `0` for the first argument.

I'd bet whoever owns the username "rate" on Github is going to get mighty tired of this thread.